data/core/units/dunefolk/Scorcher.cfg

```diff
@@ -1,18 +1,13 @@
 #textdomain wesnoth-units
 [unit_type]
+    # Below unit id may need changing to use underscores in unit ID only (not in displayed name) to fix bug #18117, as stated in former Dune Scorcher file
```

Comment:
What is 18117?

Reply:
Looks like the old GNA ID...

Reply:
I could not find reference to it in git issues